qiskit/circuit/quantumcircuit.py

```diff
@@ -940,7 +940,7 @@ def qasm(self, formatted=False, filename=None):
     def draw(self, output=None, scale=0.7, filename=None, style=None,
              interactive=False, plot_barriers=True,
              reverse_bits=False, justify=None, vertical_compression='medium', idle_wires=True,
-             with_layout=True, fold=None, ax=None, initial_state=False, cregbundle=False):
+             with_layout=True, fold=None, ax=None, initial_state=False, cregbundle=True):
```

Comment:
We are changing the default later, with https://github.com/Qiskit/qiskit-terra/pull/4595, which is blocked because a Latex drawer issue.

Reply:
Just a final word on this. cregbundle True has been the default for mpl, and I think users would be surprised if it became False. Assuming the default gets returned to True soon, it would not be an issue.

Reply:
Yes, the goal is to turn it True asap

Reply:
undone in 95416cb49